### DISCUSSION TITLE
Option to add custom DNS rexord in app like in hosts file

### DISCUSSION THREAD

**Discussion by ghostersk** [celzero/rethink-app#1744] [Ideas]
I would like to ask if it would be possible to add function where i can add custom dns record like if i would be editing the hosts file on pc? I use vpn proxy to my server and i would like to set the phone use the tunnell to acess my websites using the ip in vpn tunnell. I already have my own dns server but i cant set it differently just for phone.
Also good option could be that specific dns would be used if some proxy tunnel is ON when using Rethink

**Comment by ignoramous** [comment]
> can add custom dns record like if i would be editing the hosts file on pc

- #316 

> Also good option could be that specific dns would be used if some proxy tunnel is ON when using Rethink

A niche feature so I don't see ourselves spending time on it. Do welcome a pull request.
